non-wheel/httpsrv/httpsrv.py

```diff
@@ -268,98 +266,97 @@ def get_zendphp7_conf(name, port):
-LogMaint /www/%s/logs/error_zfcgi 10 0
+LogMaint /www/{1}/logs/access_log 10 0
+LogMaint /www/{1}/logs/error_log 10 0
+LogMaint /www/{1}/logs/error_zfcgi 10 0
 
 # Maintain Logs at 3 am (0 = midnight, 23 = 11 pm, etc)  
 # Set for an hour when the server is active (i.e. not during an IPL or backup)  
 LogMaintHour 3
 
-IncludeOptional /www/%s/conf/apache-sites/*.conf
-''' % (port, name, name, name, name, name, name)
+IncludeOptional /www/{1}/conf/apache-sites/*.conf
+'''.format(port, name)
 
 
 def get_example_vhost_conf(name, port):
-    return '''
-<VirtualHost *:%s>
-    ServerName %s.domain.com
-    DocumentRoot /www/%s/example-project/htdocs
+    return '''<VirtualHost *:{0}>
+    ServerName {1}.domain.com
+    DocumentRoot /www/{1}/example-project/htdocs
 
-    <Directory "/www/%s/example-project/htdocs">
+    <Directory "/www/{1}/example-project/htdocs">
         Options Indexes FollowSymLinks
         AllowOverride All
         Require all granted
         DirectoryIndex index.php index.html
     </Directory>
 
 
-    ErrorLog "/www/%s/example-project/logs/error_log"
-    CustomLog "/www/%s/example-project/logs/access_log" common
+    ErrorLog "/www/{1}/example-project/logs/error_log"
+    CustomLog "/www/{1}/example-project/logs/access_log" common
 </VirtualHost>
-''' % (port, name, name, name, name, name)
+'''.format(port, name)
 
 
 def create(name, conf, port):
-    path = '/www/%s' % name
-    verification_text = """
-    /www/
-        %s/
+    path = '/www/{}'.format(name)
+    verification_text = '''/www/
+    {0}/
+        conf/
+            apache-sites/
+                example-vhost.conf
+            httpd.conf
+        example-project/
             conf/
-                apache-sites/
-                    example-vhost.conf
-                httpd.conf
-            example-project/
-                conf/
-                    %s.conf
-                htdocs/
-                    index.html
-                logs/
+                {0}.conf
             htdocs/
                 index.html
             logs/
-""" % (name, name)
+        htdocs/
+            index.html
+        logs/
+'''.format(name)
 
     verification_text += "Does the above folder structure look correct? (Y/n) "
 
     verified = input(verification_text) or 'Y'
 
     if verified.lower() in ['y', 'yes']:
-        print('conf: %s' % conf)
-        print('name: %s' % name)
-        print('port: %s' % port)
-        os.mkdir('%s' % path)
-        os.mkdir('%s/conf' % path)
-        os.mkdir('%s/logs' % path)
-        os.mkdir('%s/htdocs' % path)
-        index_file = open("%s/htdocs/index.html" % path, "w+")
+        print('conf: {}'.format(conf))
+        print('name: {}'.format(name))
+        print('port: {}'.format(port))
+        os.mkdir('{}'.format(path))
+        os.mkdir('{}/conf'.format(path))
+        os.mkdir('{}/logs'.format(path))
+        os.mkdir('{}/htdocs'.format(path))
+        index_file = open("{}/htdocs/index.html".format(path), "w+")

```

Comment:
I was gonna comment before that for these that are simple concatenation, it probably makes more sense to just concatenate them:

```python
path+'/htdocs/index.html
```
vs
```python
"{}/htdocs/index.html".format(path)
```